build-tools/packages/version-tools/api-report/version-tools.api.md

```diff
@@ -52,6 +52,9 @@ export function isVersionBumpTypeExtended(type: VersionChangeType | string): typ
 // @public
 export function isVersionScheme(scheme: string): scheme is VersionScheme;
 
+// @public
+export const parseWorkspaceProtocol: (version: string) => [boolean, string];
```

Comment:
Perhaps I should export an inverse of this function instead of the prefix string, and expect people to call it instead of combining the prefix string with the range? Maybe something like:

```ts
useWorkspaceProtocol(version: string): string
```